src/expandable-section/styles.scss

```diff
@@ -124,6 +121,7 @@
 
   &-default {
     padding: awsui.$space-scaled-xs 0;
+    border-top: awsui.$border-divider-section-width solid awsui.$color-border-divider-default;
```

Comment:
The PR looks good but removing the `border-bottom` from the toggle and adding it to the content removes the border transitions